components/at_sum_members_mention/notifcation_from_members_modal.tsx

```diff
@@ -0,0 +1,150 @@
+}
+
+export interface ListItem {
+    type: ListItemType;
+    data: ChannelMember | JSX.Element;
+}
+
+const MembersContainer = styled.div`
+    flex: 1 1 auto;
+    padding: 0 4px 16px;
+    height: inherit;
+`;
+
+function NotificationFromMembersModal(props: Props) {
+    const dispatch = useDispatch();
+    const channel = useSelector(getCurrentChannel);
+    const teamUrl = useSelector(getCurrentRelativeTeamUrl);
+    const userProfiles = useSelector(getUsers);
+    const userStatuses = useSelector(getUserStatuses);
+    const displaySetting = useSelector(getTeammateNameDisplaySetting);
+
+    let featureNotificationMembers: ChannelMember[] = [];
+
+    props.userIds.forEach((userId) => {
+        const profile = userProfiles[userId];
+        const status = userStatuses[userId];
+        const displayName = displayUsername(profile, displaySetting);
+        const m = {
+            user: profile,
+            displayName,
+            status,
+        };
+        featureNotificationMembers.push(m);
+    });
+
+    featureNotificationMembers = featureNotificationMembers.map((member) => {
+        if (member.user === undefined) {
+            return {
+                user: {
+                    id: 'unknown',
+                    username: 'unknown',
+                } as UserProfile,
+                displayName: member.displayName,
+                status: member.status,
+            };
+        }
+
+        return member;
+    });
+
+    const listcp: ListItem[] = [];
+    for (let i = 0; i < featureNotificationMembers.length; i++) {
+        const member = featureNotificationMembers[i];
+        listcp.push({type: ListItemType.Member, data: member});
+    }
+
+    const openDirectMessage = async (user: UserProfile) => {
+        // we first prepare the DM channel...
+        await dispatch(openDirectChannelToUserId(user.id));
+
+        // ... qnd then redirect to it
+        browserHistory.push(teamUrl + '/messages/@' + user.username);
+    };
+
+    const loadMore = () => {};
+
+    return (
+        <Modal
+            id='notificationFromMembersModal'
+            dialogClassName='a11y__modal'
+            className='NotificationFromMembersModal'
+            backdrop={true}
+            show={props.show}
+            onHide={props.onHide}
+            onExited={() => {}}
+            role='dialog'
+            aria-modal='true'
+        >
+            <Modal.Header className='NotificationFromMembersModal__header'>
+                <h1 id='invitation_modal_title'>
+                    {`Members that requested ${props.feature}`}
+                </h1>
+                <button
+                    id='closeIcon'
+                    className='icon icon-close'
+                    aria-label='Close'
+                    title='Close'
+                    onClick={props.onHide}
+                />
+            </Modal.Header>
+            <Modal.Body>
+                <MembersContainer>
+                    <MemberList
```

Comment:
Does member list handle the case when the user is deactivated? Does it show some kind of indicator? Wondering due to @yasserfaraazkhan observation here: https://www.figma.com/file/vTjh2dmLlfYW0KeBcQLQpZ?node-id=27:128777#249628453

Reply:
@yasserfaraazkhan and @marianunez a deactivated user's profile remains. So, it still shows the user handle in the post. If the admin clicks on the handle to open a DM, then they'll find out that the user is deactivated.